### ISSUE TITLE
What is the point of the hrs_each_step variable?

### ISSUE THREAD

**Issue by StephenScinocca** [microsoft/ClimaX#40]
Hi, 

From my understanding, the hrs_each_step variable in the global_forcast_climax.yaml file is tracking how many hours are in between input samples. ERA5 for example uses hourly data, so the default is to fine tune our data using all of the available data, so hrs_each_step is 1. However, if you change hrs_each_step to any other value, you still will always have the same number on input examples as if it was 1, and the results become worse the biggest hrs_each_step gets.

I have finetuned models for different lead times (for example, 48 hours) using a hrs_each_step = 1, but if I want to generate predictions using the test set, my understanding is that when hrs_each_step is 1, there should hourly examples generated, but if hours_each_step is 24, then I would be doing daily predictions at a fixed time of the day. This is not how this currently works. 

Can you explain exactly what the hrs_each_step variable is supposed to do, if not be the number of hours in between each input sample?

**Comment by tung-nd** [comment]
Hi Stephen,

Thank you for your interest in ClimaX. Please checkout the Forecast dataset https://github.com/microsoft/ClimaX/blob/main/src/climax/pretrain/dataset.py#L67 for a better understanding of the arguments. Basically, `hrs_each_step` indicates the time difference between two consecutive recorded data points in the dataset, and is used to compute the `lead_times` vector correctly. Therefore, this value is a characteristic of a dataset and is a fixed value (1 for ERA5 and 6 for CMIP6). To specify how far the model should forecast into the future, you should use `max_predict_range` instead. For example, if you want to make a 24-hr forecast, then you should set `max_predict_range=24` and `hrs_each_step=1`.
